spec/rubocop/cop/factory_bot/create_list_spec.rb

```diff
@@ -202,6 +236,20 @@
       RUBY
     end
 
+    it 'ignores Array.new(n) when create call does have method calls ' \
+       'and repeat multiple times' do
+      expect_no_offenses(<<~RUBY)
+        Array.new(3) { create :user, repositories_count: rand }
+      RUBY
+    end
+
+    it 'ignores usage of Array.new(n) when create call does have method ' \
+       'calls and not repeat' do
+      expect_no_offenses(<<~RUBY)
+        Array.new(1) { create :user, repositories_count: rand }
+      RUBY
+    end
+
```

Comment:
```suggestion
```

Those examples are nice to have for completeness, but finding this in real code seems quite unrealistic. I would argue to adding them, as this spec is big already.